### PR TITLE
poetry & github actions

### PR DESCRIPTION
**Pull request by manhal-daaboul**
@sam-writer give this repo access to organisation secrets `JFROG_PYPI_USERNAME` and `JFROG_PYPI_PASS`

### CODE REVIEW

**Comment by sam-writer** [comment]
@manhal-daaboul I love that we're converting this to Poetry, but I don't think we need to put this on the JFrog private package repo. The way JFrog works is that it first looks for private packages, then looks in all of regular PyPi

**Comment by manhal-daaboul** [comment]
@sam-writer updated to use only public pypi

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+<!--- What types of changes does your code introduce? Replace ` ` (space) with an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
+### Reminders
+
+- [ ] I incremented the version appropriately (now in `pyproject.toml`).
+- [ ] I added my changes to the CHANGELOG.md.
```
Comment (sam-writer) [inline .github/pull_request_template.md:15]: I think we can remove this line

**Comment by sam-writer** [comment]
LGTM

**Pull request merged**